cpp/dolfin/mesh/Partitioning.h

```diff
@@ -65,6 +66,11 @@ class Partitioning
   ///     Global index for each cell
   /// @param ghost_mode
   ///     Ghost mode
+  /// @param part_processes_ratio
```

Comment:
This is now taken care of by the comm...

Reply:
Removed.